deployment/docker-postgis/Dockerfile

```diff
@@ -0,0 +1,8 @@
+#--------- Generic stuff all our Dockerfiles should start with so we get caching ------------
+FROM kartoza/postgis:9.3-2.1
+MAINTAINER Tim Sutton<tim@kartoza.com>
+
+# add entry to pg_hba.conf
+RUN echo "host    all    all    192.168.0.0/16   md5" >> /etc/postgresql/9.3/main/pg_hba.conf
```

Comment:
I think upstream postgis is already perfect and secure.
I only uses this for dev-env, and it's not gonna be used at production.
But if you think it would ease the other dev-env (and still secure), I can make a PR to upstream postgis.